src/assets/prosemirror/menu.css

```diff
@@ -0,0 +1,147 @@
+.ProseMirror-textblock-dropdown {
```

Comment:
Ah, right... forgot about that.  So you'd have to add `>>>` to absolutely everything.  Which makes me cringe a little every time I use them (kind of like adding `!important` to something).  Global it is then!